### PR TITLE
Fixed broken link

### PR DESCRIPTION
**Pull request by sureshgaikwad**
Fixed the broken link to Environment health checks

### CODE REVIEW

**Comment by sureshgaikwad** [comment]
Filed a [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1567590) for this request:

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -146,7 +146,7 @@ than 3.7, you must first upgrade incrementally (e.g., 3.5 to 3.6, then 3.6 to
 [NOTE]
 ====
 Before attempting the upgrade, follow the guidance in
-xref:../day_two_guide/environment_health_checks.html#day_two_environment_health_checks[Environment
+xref:../day_two_guide/environment_health_checks.adhoc#day-two-environment-health-checks[Environment
```
Comment (vikram-redhat) [inline upgrading/automated_upgrades.adoc]: @sureshgaikwad - hey Suresh. This should be .adoc, not adhoc. If you fix and resubmit, I can merge it.

**Pull request merged**